se2_planning/include/se2_planning/OmplReedsSheppPlanner.hpp

```diff
@@ -56,6 +56,12 @@ class OmplReedsSheppPlanner final : public OmplPlanner {
   void setParameters(const OmplReedsSheppPlannerParameters& parameters);
   void setStateValidator(std::unique_ptr<StateValidator> stateValidator);
   const StateValidator& getStateValidator() const;
+  void lockStateValidator();
+  void unlockStateValidator();
+  bool isLocked() const;
+  void updateStateSpaceBounds(const ompl::base::RealVectorBounds& bounds) override;
+  bool satisfiesStateSpaceBounds(const se2_planning::ReedsSheppState& state);
+  const ompl::base::RealVectorBounds& getStateSpaceBoundaries();
```

Comment:
save for this method. Make it const.

Reply:
Made it const and added a default implementation to OmplPlanner class in db4a618de5f69d9b9b0bee5375f1217ca36002a6.